src/connect.rs

```diff
@@ -260,7 +258,11 @@ fn set_defaults(config: &mut Config) {
     }
     if config.addr.is_none() {
         if let Some(port) = &config.port {
-            config.addr.replace(port.to_string());
+            if let Some(host) = &config.host {
+                config.addr.replace(format!("tcp://{host}:{port}"));
```

Comment:
`{TCP_PREFIX}{host}...`

Reply:
Committed a fix for this, and added `listen_and_serve`.